### DISCUSSION TITLE
增加下载资源包数据包地图光影功能

### DISCUSSION THREAD

**Discussion by computer-drive** [Meloong-Git/PCL#2524] [提问] [answered]
在下载界面增加资源包、数据包、光影、地图功能，并且推荐支持自动的将文件导入到游戏中。

**Comment by MoYuan-CN** [comment, marked as answer]
资源包见 #44（投票 #1095）
地图见 #44 #135（存在语言问题）
光影 #396（无下载源）
数据包 #871（无可用API）
